apps/admin-gui/src/app/shared/components/dialogs/add-member-group-dialog/add-member-group-dialog.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { Group, GroupsManagerService, Member, MembersManagerService } from '@perun-web-apps/perun/openapi';
+import { NotificatorService } from '@perun-web-apps/perun/services';
+import { TranslateService } from '@ngx-translate/core';
+import { SelectionModel } from '@angular/cdk/collections';
+
+export interface AddMemberGroupDialogData {
+  theme: string;
+  memberId: number;
+}
+
+@Component({
+  selector: 'app-add-member-group-dialog',
+  templateUrl: './add-member-group-dialog.component.html',
+  styleUrls: ['./add-member-group-dialog.component.scss']
+})
+export class AddMemberGroupDialogComponent implements OnInit {
+
+  constructor(private dialogRef: MatDialogRef<AddMemberGroupDialogComponent>,
+              @Inject(MAT_DIALOG_DATA) private data: AddMemberGroupDialogData,
+              private groupManager: GroupsManagerService,
+              private memberManager: MembersManagerService,
+              private notificator: NotificatorService,
+              private translate: TranslateService) {
+
+  }
+
+  theme: string;
+  loading = false;
+
+  member: Member;
+  groups: Group[] = [];
+  selection = new SelectionModel<Group>(true, []);
+
+  hideColumns = [ 'vo', 'menu' ];
+  filterValue = "";
+
+  ngOnInit(): void {
+    this.theme = this.data.theme;
+    this.loading = true;
+
+    this.memberManager.getMemberById(this.data.memberId).subscribe(member => {
+      this.member = member;
```

Comment:
If I am not mistaken you dont need the whole object member in whole dialog. Only his memberId, thats what you are using and thats what you need. So this request to the backend is useless.

Reply:
I also need member's voId that's why I have this backend request here.